docs/welcome/hello-world.md

```diff
@@ -99,7 +99,7 @@ Here we can see more of how Cpp2 makes it features work.
 **How: Simple, safe, and efficient by default.**
 
 - **Line 9: CTAD** just works, because it turns into ordinary C++ code which already supports CTAD.
-- **Lines 10-11: Automatic bounds checking** is added to `words[0]` and `words[1]` nonintrusively at the call site by default. Because it's nonintrusive, it works seamlessly with all existing container types that are `std::ssize`-aware, when you use them from safe Cpp2 code.
+- **Lines 10-11: Automatic bounds checking** is added to `words[0]` and `words[1]` nonintrusively at the call site by default. Because it's nonintrusive, it works seamlessly with all existing container types that are `std::size` and `std::ssize`-aware, when you use them from safe Cpp2 code.
```

Comment:
A few lines above:

> **Lines 8, 9, and 15: Portable C++20 code**

I'm not sure why those particular lines are called out here.  Are those the correct lines?

Reply:
I was trying to pick the three declarations, but you're right that this is confusing. I'll change it to "all lines" or similar in a separate commit.